tests/test_other.py

```diff
@@ -666,6 +666,19 @@ def test_cmake_compile_features(self):
     emscripten_features = '\n'.join([x for x in emscripten_features.split('\n') if '***' in x])
     self.assertTextDataIdentical(native_features, emscripten_features)
 
+  # Test that concat generator arguments(e.g. "-GNinja" instead of "-G Ninja") can be correctly recognized passed to cmake
+  # Windows Only
+  def test_cmake_with_concat_generator_arg(self):
+    if not utils.which('ninja'):
+      self.skipTest('Skipped concat generator argument test since ninja not found')
+    if not WINDOWS:
+      self.skipTest('Skipped concat generator argument test on non-Windows platforms')
```

Comment:
Actually, maybe these two lines are not needed, if all we are doing it checking the ninja is used, that should work on all platforms.